### PR TITLE
Implement Spring Boot

### LINKED ISSUE #1: Implement Spring Boot Base
https://projects.spring.io/spring-boot/#quick-start

### PR DESCRIPTION
**Pull request by TheNewPlan**
Resolves #1 
----------

Follow the instructions on the branches README.md

You do not download gradle to run the commands.  The purpose of gradle is so that any person can clone the repository and can build the project without having to install gradle on their machine.

So please follow the instructions mentioned and tell me if you have any issues.

I have been able to build on both of my Windows machines personally.

### CODE REVIEW

**Comment by bmartin5263** [comment]
I was able to build it using ./gradle build, however I got an error:

* What went wrong:
Execution failed for task ':findMainClass'.
> Unable to find a single main class from the following candidates [org.csc394.findadev.FindadevApplication, org.csc394.employeematch.EmployeematchApplication]

**Comment by TheNewPlan** [comment]
And this was on the bootRun step?

**Comment by bmartin5263** [comment]
Ah my apologies, it was on the build step not on the bootRun. Have you encountered this error before?

**Comment by TheNewPlan** [comment]
I did before.  A clean pull of the repo resolved it.

Gradle can't find the file I referenced above there, and is failing.  

Rather than digging thru the gradle configs just try a clean pull / build

**Comment by bmartin5263** [comment]
Gotcha, how can I do a clean pull from IntelliJ?

File: README.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ First clone the repository / branch.
...
 
 You will see it download a bunch of dependencies.  This is normal.
 
 Next, enter:
 
-Unix/OSX: './gradlew bootRun'
+Unix/OSX: ''' ./gradlew bootRun '''
 
-Windows: 'gradlew bootRun'
+Windows: '''gradlew bootRun'''
```
Comment (TheNewPlan) [inline README.md]: Well that didn't fix anything, but anyways, point is just enter the command, not the quotes.

**Comment by bmartin5263** [comment]
Should I do "Checkout as Local Branch", or do I need to use "Pull"? Sorry for all the questions I want to make sure I do it right.

**Comment by TheNewPlan** [comment]
No problem.  When I say checkout, you just checkout that remote repository as a local branch.

From there you can try running the commands.

**Comment by bmartin5263** [comment]
Success! Its running now and I checked 8080. Is it normal for it to be hanging on :bootRun 80% Executing?

**Comment by TheNewPlan** [comment]
Yeah, that's a known gradle bug.

Glad it works!  Excited to get going.  Gradle is a super helpful tool, and will only be more helpful.

**Comment by TheNewPlan** [comment]
If everyone else can check if this works for their machines, that'd be great!

**Comment by jonathan7** [comment]
This builds for me on linux just fine, I see "Hello CSC 394 Teammates"

**Comment by TheNewPlan** [comment]
Perfect, that's the intended result.

**Pull request merged**